opentelemetry-sdk/src/opentelemetry/sdk/_metrics/point.py

```diff
@@ -0,0 +1,49 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Union
+
+
+@dataclass(frozen=True)
+class Sum:
+    # NumberDataPoint attributes
+    start_time_unix_nano: int
+    time_unix_nano: int
+    value: Union[int, float]
+    # Sum attributes
+    aggregation_temporality: int
+    is_monotonic: bool
+
+
+@dataclass(frozen=True)
+class Gauge:
+    # NumberDataPoint attributes
+    # start_time_unix_nano is not added here because the proto says it is to
+    # be ignored.
+    time_unix_nano: int
+    value: Union[int, float]
+
+
+@dataclass(frozen=True)
+class Histogram:
+    # NumberDataPoint attributes
+    start_time_unix_nano: int
+    time_unix_nano: int
+    value: Union[int, float]
```

Comment:
```suggestion
    bucket_counts: Sequence[int]
    explicit_bounds: Sequence[float]
```

See https://github.com/open-telemetry/opentelemetry-proto/blob/7d4cbec4a940edc92a87b173b8d130b24085607e/opentelemetry/proto/metrics/v1/metrics.proto#L454-L469